README.md

```diff
@@ -169,70 +172,78 @@ render() {
 ### Configuration
 
 #### Slider
-| Prop name | Type | isRequired | defaultValue | Description |
-| --- | --- | --- | --- | -- |
-| `arrowRender` | `func` | :no_entry_sign: | :no_entry_sign: | A render function that will receive as props an orientation prop and an onClick callback |
-| `arrowsContainerComponent` | `func/string` | :no_entry_sign: | :no_entry_sign: | The component used to contain both arrows. Either a string to use a DOM element or a component |
-| `children` | `element/array` | :heavy_check_mark: | :no_entry_sign: | The slides to render |
-| `classes` | `object` | :no_entry_sign: | No extra classes applied to any element | Classes to apply to the Slider elements |
-| `currentSlide` | `number` | :no_entry_sign: | `0` | Current slide on the screen, if you have perPage > 1, then the current slide is the most left slide on the screen (You should not use this variable to display the index of the slide on the screen if you're using `loop={true}`). |
-| `cursor` | `string` | :no_entry_sign: | `'-webkit-grab'` | Css value of cursor when mouse is hovering the slider frame |
-| `cursorOnMouseDown` | `string` | :no_entry_sign: | `'-webkit-grabbing'` | Css value of cursor when mouse is down |
-| `duration` | `number` | :no_entry_sign: | `250` | Duration of transitions |
-| `easing` | `string` | :no_entry_sign: | `'ease-out'` | Transition function |
-| `loop` | `bool` | :no_entry_sign: | `false` | If the slides should be looping |
-| `onChangeSlide` | `func` | :heavy_check_mark: | :no_entry_sign: | Function to change the value of currentSlide. The function should expect a `number` as it's only parameter |
-| `perPage` | `number/object` | :no_entry_sign: | `1` | Amount of slides to be on the screen, if a number is passed, then that's the number of slides that will be shown, if an object with breakpoints is passed, then the component will check the size of the screen to see how many slides will be on the screen at the same time |
-| `resizeDebounce` | `number` | :no_entry_sign: | `250` | Resize debounce timer in milliseconds |
-| `rootTag` | `string` | :no_entry_sign: | `'div'` | Tag to be rendered in the root of the slider |
-| `sliderFrameTag` | `string` | :no_entry_sign: | `'ul'` | Tag to be rendered in the slider frame element |
-| `threshold` | `number` | :no_entry_sign: | `50` | Minimum of pixels to drag until the slider change the `currentSlide` |
+
+| Prop name                  | Type            | isRequired         | defaultValue                            | Description                                                                                                                                                                                                                                                                   |
```

Comment:
why this change?

Reply:
Oops... Auto format on save, i'll take it back